Packages/com.llealloo.audiolink/Runtime/Scripts/AudioLinkMiniPlayer.cs

```diff
@@ -32,11 +32,14 @@ public class AudioLinkMiniPlayer : UdonSharpBehaviour
         [Tooltip("Automatically loop track when finished")]
         public bool loop = false;
 
+        public AudioLink audioLink;
```

Comment:
Did you check if this field works with the button on the AudioLink prefab to automatically populate references to it? Would be nice if you don't have to populate this manually.

Reply:
Yes it works (just checked in sandbox project)